Makefile

```diff
@@ -7,7 +7,7 @@ include release-tools/build.make
 BUILD_PLATFORMS=windows amd64 .exe
 GOPATH ?= $(shell go env GOPATH)
 REPO_ROOT = $(CURDIR)
-BUILD_DIR = build
+BUILD_DIR = bin
```

Comment:
after this change, the binary will be generated under bin/?

Reply:
yes, however I believe that this BUILD_DIR is only used for `csi-proxy-api-gen` and not `csi-proxy`